app/views/ChargeSummary.scala.html

```diff
@@ -324,7 +339,7 @@ <h1 class="govuk-heading-xl">
         }
     }
 
-    @if(viewModel.noIsAgentAndRemainingToPayWithNoCodingOutEnabledAndIsPayeSelfAssessment){
+    @if(viewModel.noIsAgentAndRemainingToPayWithNoCodingOutEnabledAndIsPayeSelfAssessment && !viewModel.isReviewAndReconcileDebit){
```

Comment:
I guess we can resolve all logic on the level of view model? so extra condition added can be moved within noIsAgentAndRemainingToPayWithNoCodingOutEnabledAndIsPayeSelfAssessment property ?